src/index.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import Export from './routers/export';
+import Import from './routers/import';
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(Export);
+app.use(Import);
+app.listen(3000, () => {
```

Comment:
would be nice if the port was configurable :D

Reply:
I will provide it.